R/read_and_process_embrace_plus.R

```diff
@@ -23,7 +23,11 @@ read_and_process_embrace_plus <- function(zipfile, tz = Sys.timezone()) {
 #' @param data object from read_e4 function
 process_embrace_plus <- function(data) {
 
-  eda_filt <- process_eda(data$EDA)
+  # omitting NAs: TBD
+  # the Embrace Plus aggregated files have a lot of NAs,
+  # for example when the device wasn't able to record anything
+  # we need to decide how to handle these NAs
+  eda_filt <- process_eda(na.omit(data$EDA))
```

Comment:
As you know I'm no expert when it comes to analysing this data, but the`butter_lowpass_filter` doesn't work the same as it does with other devices when there are a lot of NAs. The aggregated Embrace Plus data also has "data" present when the device wasn't worn. When I removed the NAs the flow seemed to work fine again, but would be good to double check what the implications are